package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-branch",
-  "version": "5.6.2",
+  "version": "5.7.0",
```

Comment:
For upping ios requirements, do we usually bump the major? or is 9->11 not disruptive enough to warrant?